project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/components/window/popup.less

```diff
@@ -27,12 +27,12 @@
     transition: @transition transform;
     border-radius: @radius;
 
-    .is-no-flex &, .is-safari &, .is-flex-popup-height-issue-detected & {
+    .is-safari &, .is-flex-popup-height-issue-detected & {
```

Comment:
Is the class `is-safari` still necessary?

Reply:
really dont know ... but will be checked later during refactoring

Reply:
Okay, let's refactor this later.